src/source/PeerConnection/SessionDescription.c

```diff
@@ -420,12 +438,20 @@ STATUS populateSingleMediaSection(PKvsPeerConnection pKvsPeerConnection, PKvsRtp
             currentFmtp = fmtpForPayloadType(payloadType, pRemoteSessionDescription);
         }
     }
+
+    if (RTC_CODEC_H265 == pRtcMediaStreamTrack->codec) {
+        payloadType = DEFAULT_PAYLOAD_H265;
+    }
```

Comment:
Why do we need to do this? It looks like we do this again in LN 452-452.

Reply:
Removed